content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -1,72 +1,231 @@
-====================================================================
-When Should you Use Packages, Units of Measure or Special Packaging?
-====================================================================
+==========================================
+Units of measure, packages, and packagings
+==========================================
 
-Units of Measure
+In Odoo, there are a variety of ways to specify the amount of products being bought, stocked, and
+sold. *Units of measure*, *packages*, and *packagings* are all available to streamline inventory
+flows, allowing for a variety of configurations for products as they enter and leave the warehouse.
+
+Units of measure
 ================
 
-Units of measure specify the unit used to handle a product. In Odoo, you
-have the possibility to specify the unit of measure in which you manage
-your stock and the one which is used when purchasing the product to your
-supplier.
+*Units of measure (UoM)* refers to different measurable quantities used to handle products, such as
+units, weight, time, size, etc. Different :abbr:`UoMs (Units of Measure)` for weight can be kilos,
+pounds, ounces, grams, etc.
+
+In Odoo, stock is easily managed and products are more efficiently purchased from suppliers by
+specifying different :abbr:`UoM (Units of Measure)` for both purchasing and selling.
+
+.. image:: usage/uom-on-product-form.png
+   :align: center
+   :alt: Specify unit of measure for selling a product vs purchasing.
 
-.. image:: usage/usage_01.png
-    :align: center
+Once a product has a default :guilabel:`Unit of Measure` and :guilabel:`Purchase Unit of Measure`
+set on the product form, Odoo automatically converts the different units in the product's
+purchase/sales orders and the orders' respective delivery orders/receipts.
 
-The *conversion* between the different units of measure is done
-automatically. The only condition is that all the units have to be in
-the *same category* (Unit, Weight, Volume, Length,...)
+The only condition is that all of the units have to be in the *same category* (unit, weight, volume,
+length, etc.).
 
-For example, if I have the following reordering rule for the egg and I
-run the scheduler, the quantity added in the automatically generated
-purchase order will be in dozens but what will enter the stock will be
-units.
+.. example::
+   The product, `Rope` has its :guilabel:`Unit of Measure` field set to feet (:guilabel:`ft`) and
+   its :guilabel:`Purchase UoM` set to centimeters (:guilabel:`cm`). When creating a :abbr:`PO
+   (Purchase Order)` for the product, the :guilabel:`Purchase UoM (Unit of Measure)`, centimeters
+   (:guilabel:`cm`), appears on the form to measure the quantity of rope the company buys from the
```

Comment:
I'm suddenly confused again on why we're putting a guilabel on cm, ft in parantheses. Here's the UI:
![image](https://github.com/odoo/documentation/assets/23611711/e78bf7e1-feac-4452-b757-85a322a4ac9a)

Since we're not putting a guilabel on Rope or 16.4, I'm confused again :sweat_smile:

Reply:
yea these are the values of inputs (not headings, labels, etc.) so they'd receive single back ticks, not guis

Reply:
I see - I misunderstood. My point of thinking was I thought these units were selections from the **Unit of Measure** field on the product form, and not a literal input of text "ft".

Reply:
Thanks for reminding me, Sam (: You made a good point about checking exactly what the UI displays, so I rewrote the section to match quantities that appears on the product form and PO.
![image](https://github.com/odoo/documentation/assets/23611711/cbce739c-81c7-4994-9b0b-0936c1b8f7ba)

```suggestion
   On the product form for `Rope`, the following fields are set as:

   - :guilabel:`Unit of Measure` in `ft` (feet), and
   - :guilabel:`Purchase UoM` in `cm` (centimeters).

   Because the vendor sells rope in `cm`, the purchase :guilabel:`UoM ` is used to represent the
   quantity on the :abbr:`PO (Purchase Order)`.
```
![image](https://github.com/odoo/documentation/assets/23611711/56ebabe7-a34c-41c5-a2c4-c42c1d0b667a)